cypress/integration/pattern/pattern.test.js

```diff
@@ -0,0 +1,174 @@
+    cy.get('[data-cy=Доступний]').contains(
+      config.tableHeadRowTitles.patterns[3]
+    );
+    cy.get('[data-cy=Дії]').should('be.visible');
+    cy.get('[data-cy=Дії]').contains(config.tableHeadRowTitles.patterns[4]);
+
+    cy.get('[data-cy=add-pattern]').click();
+    cy.get('[data-cy=handmade]').should('be.visible');
+    cy.get('[data-cy=handmade]').click();
+    cy.get('[data-cy=available]').should('be.visible');
+    cy.get('[data-cy=available]').click();
+    cy.get('[data-cy=go-back-btn]').should('be.visible');
+    cy.get('[data-cy=save-btn]').should('be.visible');
+    cy.get('[data-cy=save-btn]').contains(config.buttonTitles.SAVE_TITLE);
+    cy.get('[data-cy=pattern-image]').should('be.visible');
+    cy.get('[data-cy=add-photo]').should('be.visible');
+    cy.get('[data-cy=add-photo]').contains('Завантажити');
+    cy.get('[data-cy=material').should('be.visible');
+    cy.get('[data-cy=material').contains(config.labels.pattern.material);
+    cy.get('[data-cy=ua-name]').should('be.visible');
+    cy.get('[data-cy=ua-name]').contains('Назва');
+    cy.get('[data-cy=ua-description]').should('be.visible');
+    cy.get('[data-cy=ua-description]').contains('Опис');
+    cy.get('[data-cy=ua-tab]').should('be.visible');
+    cy.get('[data-cy=en-tab]').should('be.visible');
+    cy.get('[data-cy=en-tab]').click();
+    cy.get('[data-cy=en-name]').should('be.visible');
+    cy.get('[data-cy=en-name]').contains('Назва');
+    cy.get('[data-cy=en-description]').should('be.visible');
+    cy.get('[data-cy=en-description]').contains('Опис');
+  });
+
+  it('should check validation', () => {
+    cy.get('[data-cy=add-pattern]').click();
+    cy.get('[data-cy=save-btn]').click();
+    cy.get('[data-cy=material-error]').contains(
+      config.patternErrorMessages.PATTERN_ERROR_MESSAGE
+    );
+    cy.get('[data-cy=ua-name-error]').contains(
+      config.patternErrorMessages.PATTERN_ERROR_MESSAGE
+    );
+    cy.get('[data-cy=ua-description-error]').contains(
+      config.patternErrorMessages.PATTERN_ERROR_MESSAGE
+    );
+    cy.get('[data-cy=en-tab]').click();
+    cy.get('[data-cy=en-name-error]').contains(
+      config.patternErrorMessages.PATTERN_ERROR_MESSAGE
+    );
+    cy.get('[data-cy=en-description-error]').contains(
+      config.patternErrorMessages.PATTERN_ERROR_MESSAGE
+    );
+    cy.get('[data-cy=material').type('a');
+    cy.get('[data-cy=en-name]').type('a');
+    cy.get('[data-cy=en-name-error]').contains(
+      config.patternErrorMessages.PATTERN_VALIDATION_ERROR
+    );
+    cy.get('[data-cy=en-description]').type('a');
+    cy.get('[data-cy=en-description-error]').contains(
+      config.patternErrorMessages.PATTERN_VALIDATION_ERROR
+    );
+    cy.get('[data-cy=ua-tab]').click();
+    cy.get('[data-cy=ua-name]').type('a');
+    cy.get('[data-cy=ua-name-error]').contains(
+      config.patternErrorMessages.PATTERN_VALIDATION_ERROR
+    );
+    cy.get('[data-cy=ua-description]').type('a');
```

Comment:
maybe it makes sense to move 'a' value to variable

Reply:
fixed